docs/framework/reflection-and-codedom/collectible-assemblies.md

```diff
@@ -0,0 +1,85 @@
+
+- A delegate to a static method that is defined in a module of the assembly.
+
+If only one item from this list exists for only one type or one method in the assembly, the runtime cannot unload the assembly.
+
+> [!NOTE]
+> The runtime does not actually unload the assembly until finalizers have run for all items in the list.
+
+For purposes of tracking lifetime, a constructed generic type such as `List<int>` (in C#) or `List(Of Integer)` (in Visual Basic) that is created and used in the generation of a collectible assembly is considered to have been defined either in the assembly that contains the generic type definition or in an assembly that contains the definition of one of its type arguments. The exact assembly that is used is an implementation detail and subject to change.
+ 
+## Restrictions on collectible assemblies
+
+The following restrictions apply to collectible assemblies: 
+
+- **Static references**   
+  Types in an ordinary dynamic assembly cannot have static references to types that are defined in a collectible assembly. For example, if you define an ordinary type that inherits from a type in a collectible assembly, a <xref:System.NotSupportedException> exception is thrown. A type in a collectible assembly can have static references to a type in another collectible assembly, but this extends the lifetime of the referenced assembly to the lifetime of the referencing assembly.
+
+- **COM interop**   
+   No COM interfaces can be defined within a collectible assembly, and no instances of types within a collectible assembly can be converted into COM objects. A type in a collectible assembly cannot serve as a COM callable wrapper (CCW) or runtime callable wrapper (RCW). However, types in collectible assemblies can use objects that implement COM interfaces.
+
+- **Platform invoke**   
+   Methods that have the <xref:System.Runtime.InteropServices.DllImportAttribute> attribute will not compile when they are declared in a collectible assembly. The <xref:System.Reflection.Emit.OpCodes.Calli?displayProperty=fullname> instruction cannot be used in the implementation of a type in a collectible assembly, and such types cannot be marshaled to unmanaged code. However, you can call into native code by using an entry point that is declared in a non-collectible assembly.
+ 
+- **Marshaling**   
+   Objects (in particular, delegates) that are defined in collectible assemblies cannot be marshaled. This is a restriction on all transient emitted types.
+
+- **Assembly loading**   
+   Reflection emit is the only mechanism that is supported for loading collectible assemblies. Assemblies that are loaded by using any other form of assembly loading cannot be unloaded.
+ 
+- **Context-bound objects**   
+   Context-static variables are not supported. Types in a collectible assembly cannot extend ContextBoundObject. However, code in collectible assemblies can use context-bound objects that are defined elsewhere.
```

Comment:
This entry doesn't render correctly (with a `<br>`, at least on GitHub). I think it's because the trailing spaces are U+00A0 NO-BREAK SPACE, not normal spaces.